modules/server/src/network/resolvers/aggregations.ts

```diff
@@ -61,9 +63,9 @@ const fetchData = async <SuccessType>(
 			}
 		}
 		return failure(CONNECTION_STATUS.ERROR, `Unknown error`);
-	} finally {
-		console.log(`Fetch data completing for ${query.url}`);
 	}
+	// TS would like a return value outside of try/catch handling
+	return failure(CONNECTION_STATUS.ERROR, `Unknown error`);
```

Comment:
even though results are handled in the `try/catch` block, TS doesn't like the implicit undefined from the function.
It is a tsconfig setting, but I think it's best to return something rather than tweak the project setting.

TS also doesn't like nothing returned in a `finally` block, and returning something here would override the returned values in the `catch` block